test/util/util.go

```diff
@@ -216,7 +216,8 @@ func WaitForInstanceReconciledGeneration(client v1beta1servicecatalog.Servicecat
 				return false, fmt.Errorf("error getting Instance %v/%v: %v", namespace, name, err)
 			}
 
-			if instance.Status.ReconciledGeneration == reconciledGeneration {
+			if instance.Status.ObservedGeneration >= reconciledGeneration &&
```

Comment:
Should change the name of the `WaitForInstanceReconciledGeneration` function and its `reconciledGeneration` parameter since it is now waiting for the generation to be processed rather than waiting for a particular reconciled generation.